examples/position.rs

```diff
@@ -0,0 +1,129 @@
+extern crate cursive;
+
+use cursive::Cursive;
+use cursive::event::Key;
+use cursive::menu::MenuTree;
+use cursive::traits::*;
+use cursive::view::Position;
+use cursive::views::Dialog;
+use cursive::views::LayerPosition;
+use cursive::views::TextView;
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+/// Moves top layer by the specifyed amount
+fn move_top(c: &mut Cursive, x_in: isize, y_in: isize) {
+    // Step 1. Get the current position of the layer.
+    let s = c.screen_mut();
+    let l = LayerPosition::FromFront(0);
+
+    let (x, y) = s.offset().pair();
+
+    // Step 2. add the specifed amount
+    // (unsigned math in Rust is a mess.)
+    let x = if x_in < 0 {
+        x - (-x_in) as usize
+    } else {
+        x + x_in as usize
+    };
+    let y = if y_in < 0 {
+        y - (-y_in) as usize
+    } else {
+        y + y_in as usize
+    };
+
+    // convert the new x and y into a position
+    let p = Position::absolute((x, y));
+
+    // Step 3. Apply the new position
+    s.reposition_layer(l, p);
+}
+
+fn main() {
+    let mut siv = Cursive::new();
+    siv.set_fps(60);
+
+    // We'll use a counter to name new files.
+    let counter = AtomicUsize::new(1);
+
+    // This is a direct copy of the menubar example.
```

Comment:
Maybe we should keep this example minimal, and focused on moving a layer. I think a simple window that can be moved around should be enough.